src/templates/layouts/vetCenter/ExpandableOperatingStatus.tsx

```diff
@@ -0,0 +1,63 @@
+import { FacilityOperatingStatusFlags } from '@/types/drupal/node'
+
+const statusAlerts: Record<
+  FacilityOperatingStatusFlags,
+  { status: 'info' | 'warning' | 'error'; text: string }
+> = {
+  notice: { status: 'info', text: 'Facility notice' },
+  limited: { status: 'info', text: 'Limited services and hours' },
+  temporary_closure: {
+    status: 'warning',
+    text: 'Temporary facility closure',
+  },
+  temporary_location: { status: 'warning', text: 'Temporary location' },
+  virtual_care: { status: 'warning', text: 'Virtual care only' },
+  coming_soon: { status: 'warning', text: 'Coming soon' },
+  closed: { status: 'warning', text: 'Facility closed' },
+}
+
+interface ExpandableOperatingStatusProps {
+  operatingStatusFlag: FacilityOperatingStatusFlags
+  operatingStatusMoreInfo: string | null
+}
+
+/**
+ * Returns an alert with the appropriate operating status.
+ * Returns null if the operating status is 'normal' or invalid.
+ */
+export const ExpandableOperatingStatus = ({
+  operatingStatusFlag,
+  operatingStatusMoreInfo,
+}: ExpandableOperatingStatusProps) => {
+  const statusAlert = statusAlerts[operatingStatusFlag]
+  if (statusAlert) {
+    const { status, text } = statusAlert
+    const style = { maxWidth: '303px' }
```

Comment:
Oh curious. Is there a design token we can use for this value, perchance? 🤔

Reply:
Hmm, I was just copying it verbatim, but there's probably a close-enough token value we could use